test/test_cpu/test_alg_ext.py

```diff
@@ -1,37 +1,30 @@
-import copy
-import shutil
-import sys
-import unittest
-
-from parameterized import parameterized
-
-sys.path.insert(0, "../..")
-
 from auto_round import AutoRound
 
+from ..helpers import opt_name_or_path, qwen_name_or_path
```

Comment:
how about providing a mapping dict or funcitons,  for qwen,  we need qwen-0.6, qwen-8B both